x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.test.tsx

```diff
@@ -269,86 +290,11 @@ describe.skip('unified data table', () => {
         fireEvent.click(screen.getAllByTestId('docTableExpandToggleColumn')[0]);
 
         await waitFor(() => {
-          expect(screen.getByTestId('timeline:details-panel:flyout')).toBeVisible();
+          expect(openFlyoutMock).toHaveBeenCalledTimes(1);
         });
       },
       SPECIAL_TEST_TIMEOUT
     );
-
-    it(
```

Comment:
What happened to these test?

Reply:
One test was testing the same thing but the implementation, I find it better to test just the behaviour as done in previous test.

Second test was valid for old flyout. Now that expandable flyout is out. This test is no longer needed.